stock_putaway_product_form/models/product_template.py

```diff
@@ -0,0 +1,62 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    product_tmpl_putaway_ids = fields.One2many(
+        comodel_name="stock.fixed.putaway.strat",
+        inverse_name="product_tmpl_id",
+        string="Product putaway strategies by product",
+    )
+
+    product_putaway_categ_ids = fields.Many2many(
```

Comment:
I think that this must be displayed in category view... So we can save a lot code lines...